store/tx_manager.go

```diff
@@ -476,30 +476,29 @@ func (txm *EthTxManager) handleUnconfirmed(
 	txat *models.TxAttempt,
 	blkNum uint64,
 ) (*TxReceipt, error) {
-	bumpable := tx.Hash == txat.Hash
-	pastThreshold := blkNum >= txat.SentAt+txm.config.EthGasBumpThreshold()
-	if bumpable && pastThreshold {
-		logger.Debugw(
-			fmt.Sprintf("Unconfirmed TX %d attempt %s, bumping gas", txat.TxID, txat.Hash.Hex()),
-			"txHash", txat.Hash.String(),
-			"txid", txat.TxID,
-			"gasPrice", txat.GasPrice.String(),
-			"from", tx.From.Hex(),
-			"blkNum", blkNum,
-			"sentAt", txat.SentAt,
-		)
-		return nil, txm.bumpGas(txat, blkNum)
+	if tx.Hash == txat.Hash {
```

Comment:
If `tx.Hash == txat.Hash` then it is the latest attempt and you do not want to exit early. Seems like this should be a `!=`.

Reply:
Ooops, thanks for picking this up, I meant to make that change and forgot.